addons/website_sale/models/digest.py

```diff
@@ -23,7 +23,10 @@ def _compute_kpi_website_sale_total_value(self):
                 ('website_id', '!=', False),
                 ('company_id', '=', company.id)
             ])
-            record.kpi_website_sale_total_value = sum(confirmed_website_sales.mapped('amount_total'))
+            record.kpi_website_sale_total_value = sum(
+                sale.currency_id._convert(sale.amount_total, company.currency_id, company, sale.date_order)
```

Comment:
FYI, there is a field `currency_rate` which stores the conversion rate from the order currency to the company currency which could have been used instead of the full conversion logic.